metron-platform/metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro-assembly/src/main/assembly/assembly.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+  Licensed to the Apache Software
+	Foundation (ASF) under one or more contributor license agreements. See the
+	NOTICE file distributed with this work for additional information regarding
+	copyright ownership. The ASF licenses this file to You under the Apache License,
+	Version 2.0 (the "License"); you may not use this file except in compliance
+	with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software distributed
+	under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+	OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+  the specific language governing permissions and limitations under the License.
+  -->
+
+<assembly>
```

Comment:
Our extensions represent a merging of our assembly based packaging and nifi's nar packaging and projects.  It is more the model of the processor nars with the assembly added in than anything else.

In a nar processor project created by the archetype, the structure is much as it is here.  I modeled after that and did not think to refactor it.   There is a parent, a nar module and a module for the actual processor code.  That I did was pull the assembly out of where metron would have it in the parser module and make it it's own module at the top level.  This made sense to me because the assembly was working with the products of the other two modules. 

I wanted to keep it simple straight forward.  If the assembly had to assemble the bundle results, then it needed to be in the bundle project not the parser project..... and things get not so simple.